numpy/core/arrayprint.py

```diff
@@ -408,7 +395,8 @@ def wrapper(self, *args, **kwargs):
 @_recursive_guard()
 def array2string(a, max_line_width=None, precision=None,
                  suppress_small=None, separator=' ', prefix="",
-                 style=np._NoValue, formatter=None):
+                 style=np._NoValue, formatter=None, threshold=None,
+                 edgeitems=None, sign=None):
```

Comment:
Can we add these extra arguments via **kwargs**, rather than supporting them positionally? It makes things a little easier if we expose yet more arguments

Reply:
Might be nice to deprecate passing these positionally too